memcache/memcache.go

```diff
@@ -711,3 +729,244 @@ func (c *Client) incrDecr(verb, key string, delta uint64) (uint64, error) {
 	})
 	return val, err
 }
+
+type MetaSetItem struct {
+	Key    string
+	Flags  MetaSetFlags
+	Length int32
```

Comment:
Is length required here ? Cant we infer this from value array ?

Reply:
Good catch, forgot to remove- done!